Does not recognize Poetry 1.2.0 `dev` section

In Poetry 1.2.0, `[tool.poetry.dev-dependencies]` is now deprecated and is equivalent to `[tool.poetry.group.dev.dependencies]`. flake8-requirements does not appear to pick up the latter, as I have the following configuration:

```toml
[tool.poetry.group.dev.dependencies]
#...
Flake8-pyproject = "^1.1.0"
flake8-requirements = "^1.5.2"
#...
pyinstaller = "^5.3"
#...

[tool.flake8]
#...
known-modules = "...,pyinstaller:[PyInstaller],..."
#...
```

And I still get these error messages:
```
packaging/pyinstaller.py:9:1: I900 'PyInstaller' not listed as a requirement
packaging/pyinstaller.py:15:5: I900 'PyInstaller' not listed as a requirement
```

I'm running flake8-requirements 1.6.1.

--- a/src/flake8_requirements/checker.py
+++ b/src/flake8_requirements/checker.py
@@ -18,7 +18,7 @@ from .modules import STDLIB_PY2
 from .modules import STDLIB_PY3
 
 # NOTE: Changing this number will alter package version as well.
-__version__ = "1.6.1"
+__version__ = "1.6.2"
 __license__ = "MIT"
 
 LOG = getLogger('flake8.plugin.requirements')
@@ -566,6 +566,10 @@ class Flake8Checker(object):
             poetry.get('dependencies', ())))
         requirements.extend(parse_requirements(
             poetry.get('dev-dependencies', ())))
+        # Collect dependencies from groups (since poetry-1.2).
+        for _, group in poetry.get('group', {}).items():
+            requirements.extend(parse_requirements(
+                group.get('dependencies', ())))
         return requirements
 
     @classmethod

--- a/test/test_poetry.py
+++ b/test/test_poetry.py
@@ -51,3 +51,18 @@ class PoetryTestCase(unittest.TestCase):
             checker = Flake8Checker(None, None)
             mods = checker.get_mods_3rd_party()
             self.assertEqual(mods, ModuleSet({"tools": {}, "dev_tools": {}}))
+
+    def test_3rd_party_groups(self):
+        content = "[tool.poetry.dependencies]\ntools='1.0'\n"
+        content += "[tool.poetry.group.dev.dependencies]\ndev-tools='1.0'\n"
+
+        with mock.patch(builtins_open, mock.mock_open()) as m:
+            m.side_effect = (
+                IOError("No such file or directory: 'setup.py'"),
+                IOError("No such file or directory: 'setup.cfg'"),
+                mock.mock_open(read_data=content).return_value,
+            )
+
+            checker = Flake8Checker(None, None)
+            mods = checker.get_mods_3rd_party()
+            self.assertEqual(mods, ModuleSet({"tools": {}, "dev_tools": {}}))

